robottelo/vm.py

```diff
@@ -372,6 +372,56 @@ def execute_foreman_scap_client(self, policy_id=None):
             raise VirtualMachineError(
                 'Failed to execute foreman_scap_client run.')
 
+    def configure_rhai_client(self, activation_key, org, rhel_distro):
```

Comment:
What is the necessity for moving this code to vm.py?  Is it not good to have this code separately in a rhai module since it will not be used anywhere else in robottelo?